containers/unit_tests/TestWithoutInitializing.hpp

```diff
@@ -24,6 +24,28 @@
 
 #include <../../core/unit_test/tools/include/ToolTestingUtilities.hpp>
 
+//! Get fence event message for @c ZeroMemset.
+template <typename execution_space>
+std::string fence_event_message_for_zeromemset(const execution_space&) {
+  return "HostSpace fence";
+}
+
+/**
+ * @name Some tests are skipped for @c CudaUVM memory space.
+ *
+ * @todo To be revised according to the future of @c KOKKOS_ENABLE_CUDA_UVM .
+ */
+///@{
```

Comment:
Not requiring changes but these are 5 lines of comment (not even counting the closing comment) when one single line would do and we do not really use Doxygen so chances are it would not even generate correct documentation after other edit the file.

Reply:
I've used `///` instead of `/** ... */`. Tell me if it's good for you ;)